README.md

```diff
@@ -5,7 +5,7 @@
 
 > ⚠️ This project is experimental and not suitable for production use. ⚠️
```

Comment:
```suggestion
> ⚠️ Temporal CLI's API is still a subject for changes. ⚠️
```

Reply:
"The APIs used by Temporal CLI are subject to change."

Or, if the warning should be a bit stronger, "Be aware that the APIs used by Temporal CLI are subject to change."

Reply:
@djmagee  I think the CLI command names/format is the "API" that Roey/Ruslan are talking about.